packages/client/components/ActionMeetingAgendaItems.tsx

```diff
@@ -61,7 +61,7 @@ const ActionMeetingAgendaItems = (props: Props) => {
   if (!agendaItem) return null
   const {content, teamMember} = agendaItem
   const {picture, preferredName} = teamMember
-  const allowedThreadables: DiscussionThreadables[] = endedAt ? [] : ['comment', 'task']
+  const allowedThreadables: DiscussionThreadables[] = endedAt ? [] : ['comment', 'task', 'poll']
```

Comment:
Yeah, I was wondering about that. I personally would keep it that way as it's a bit more future proof (maybe at some point backend will send these?). A helper function is kind of like using a singleton, although I agree, it'd be nice to have fewer props. Any thoughts on this @mattkrick?

Reply:
it's a hard problem because the prop is a function of domain + local state. (e.g. is the meeting ended? is it a poker meeting? is someone already adding a task?)
We also don't want the discussion thread to depend on any meeting state because in the future a discussion can live outside a meeting, so we need to do those calcs in a parent.

it's used in 12 files, 6 of those files only pass the props through.

- we could put them in context, but IMO that'd do more harm than good because a dev would still have to climb up the tree to see where the context is set. context always feels a little magical as it's just this state store floating out there
- we could store it in the client cache, but then we'd have to write updaters for things like when the meeting ends & the discussion thread becomes read-only
- we could pass in an object with some methods that are fancier than `Array.includes`


Not sure any of those are substantially better, but if you have a strong opinion let me know!

Reply:
Let's leave it as is then! None of these options sounds like a better solution IMO.